notebook/utils.py

```diff
@@ -98,22 +115,15 @@ def is_file_hidden_win(abs_path, stat_res=None):
     if os.path.basename(abs_path).startswith('.'):
         return True
 
-    # check that dirs can be listed
-    if os.path.isdir(abs_path):
-        # can't trust os.access on Windows because it seems to always return True
-        try:
-            os.stat(abs_path)
-        except OSError:
-            # stat may fail on Windows junctions or non-user-readable dirs
-            return True
```

Comment:
Removed checking for a broken symlink as I think it's better to let the user know and have them deal with it rather than pretend it doesn't exist